wolox-training/mr-rails
# Owners

# Summary
* Adding owner to articles and comments

# Test plan
1. Login to the blog
2. Create a new article
3. Create new comments
4. Edit and delete your article and comments
5. It should not be possible for you to edit/delete articles and comments not created by you

# Trello card
https://trello.com/c/CptuyBS4/28-agregarle-creador-a-los-articulos-y-comentarios
